src/messages/embed.ts

```diff
@@ -8,6 +8,22 @@ import type {
 	APIEmbedThumbnail,
 	APIEmbedVideo,
 } from 'discord-api-types/v9';
+import ow from 'ow';
+
+const embedFieldPredicate = ow.object.exactShape({
+	name: ow.string.minLength(1).maxLength(256),
+	value: ow.string.minLength(1).maxLength(1024),
+	inline: ow.optional.boolean,
+});
+
+const embedFieldsArrayPredicate = ow.array.ofType(embedFieldPredicate);
+
+function validateFieldLength(amountAdding: number, fields: APIEmbedField[]): void {
+	if (amountAdding + fields.length > 25)
+		throw RangeError(
```

Comment:
ow validates this, `ow(fields.length + amountAdding, 'field amount', ow.number.maxWhatItIs(25))`